tools/ci/bazelrc

```diff
@@ -0,0 +1,30 @@
+# Limit resources since Travis GCE VMs have 2 cores and 7.5 GB RAM.
+build --local_resources=4000,2,1.0
+build --worker_max_instances=2
+
+ # Ensure sandboxing is on to increase hermeticity.
+build --spawn_strategy=sandboxed
+build --worker_sandboxing
+
+ # Ensure the PATH env var from our virtualenv propagates into tests, which is
+# no longer on by default in Bazel 0.21.0 and possibly again in the future.
+# We set this flag for "build" since "test" inherits it, but if we don't set
+# it for build too, this causes a rebuild at test time.
+#
+build --action_env=PATH
+
+ # Set up caching on local disk so incremental builds are faster.
```

Comment:
Is `.cache` persisted across travis builds?

Reply:
@eric-wieser Yes.